### PR TITLE
added 2 comments

### PR DESCRIPTION
**Pull request by ShubhamTathe56**
hii teams...

plese review the code and give sujjections on it ..

thanks 
shubham56

### CODE REVIEW

File: src/test/java/ASSIGNMENT/Assignment_1.java [hunk 1]
```diff
@@ -39,17 +39,20 @@ public void setUp()
 	//1.For RadioButton
 	public void radiobutton()
 	{
+		System.out.println("this is 1st comments");
```
Comment (shubhamtathe97) [inline src/test/java/ASSIGNMENT/Assignment_1.java]: add something here..........

File: src/test/java/ASSIGNMENT/Assignment_1.java [hunk 2]
```diff
@@ -39,17 +39,20 @@ public void setUp()
...
         Assert.assertEquals(true, radio2.isSelected()); 
+        
          System.out.println(" radio buttton is selected");
          }
 	
 	@Test
 	//2.For Drop Down Handling
 	public void dropDown()
 	{
+		System.out.println("this is 2nd comments");
```
Comment (shubhamtathe97) [inline src/test/java/ASSIGNMENT/Assignment_1.java]: also check this......

**Review by shubhamtathe97 — COMMENTED** [review]
please correct this mention changes....

File: src/test/java/ASSIGNMENT/Assignment_1.java (hunk 1 above)
Comment (ShubhamTathe56) [reply src/test/java/ASSIGNMENT/Assignment_1.java]: made the changes......

File: src/test/java/ASSIGNMENT/Assignment_1.java (hunk 2 above)
Comment (ShubhamTathe56) [reply src/test/java/ASSIGNMENT/Assignment_1.java]: made the changes......

**Comment by shubhamtathe97** [comment]
ok u can merge fec-01 with master..........

**Pull request merged**